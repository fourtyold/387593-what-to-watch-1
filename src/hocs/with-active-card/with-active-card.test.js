import React from "react";
import renderer from "react-test-renderer";
import withActiveCard from "./with-active-card.js";

const MockComponent = () => <div />;
const MockComponentWrapped = withActiveCard(MockComponent);

const options = {
  film: {
    image: {
      name: `fantastic-beasts-the-crimes-of-grindelwald`,
      extension: `jpg`
    },
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    page: `movie-page.html`,
    preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  cardHeaderClickHandler: () => {},
};

it(`With-active-card renders correctly`, () => {
  const tree = renderer.create(<MockComponentWrapped
    film={options.film}
    cardHeaderClickHandler={options.cardHeaderClickHandler}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
