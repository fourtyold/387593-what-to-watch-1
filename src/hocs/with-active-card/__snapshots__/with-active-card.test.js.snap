// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With-active-card renders correctly 1`] = `<div />`;
